test/functional/services/renderable.ts

```diff
@@ -26,10 +26,17 @@ export class RenderableService extends FtrService {
    */
   public async waitForRender(count: number = 1): Promise<void> {
     this.log.debug(`Renderable.waitForRender for ${count} elements`);
+    // exit if there are no objects expected
+    if (count === 0) {
+      return;
+    }
     await this.retry.try(async () => {
       const completedElements = await this.find.allByCssSelector(RENDER_COMPLETE_SELECTOR);
       if (completedElements.length < count) {
-        const pendingElements = await this.find.allByCssSelector(RENDER_COMPLETE_PENDING_SELECTOR);
+        const pendingElements = await this.find.allByCssSelector(
+          RENDER_COMPLETE_PENDING_SELECTOR,
+          2500
+        );
```

Comment:
Since `pendingElements` maybe zero for a case when dashboard has only view items, we can decrease timeout from default 10000 ms to 2500 ms and still be properly waiting because we later verify if there are still any loading items: `const stillLoadingElements = await this.find.allByCssSelector(DATA_LOADING_SELECTOR, 1000);`